admin/app/components/solidus_admin/orders/cart/component.js

```diff
@@ -0,0 +1,129 @@
+import { Controller } from "@hotwired/stimulus"
+import { useClickOutside, useDebounce } from "stimulus-use"
+
+const LOADING = "Loading..."
+const INITIAL = "Type to search"
+const EMPTY = "No results"
+const QUERY_KEY = "q[name_or_variants_including_master_sku_cont]"
```

Comment:
In the current admin we are able to change the query in order to reuse the variant select in other areas of the admin. We should make this possible here as well.